inc/shortcodes/class-facebook.php

```diff
@@ -21,6 +21,11 @@ public static function get_shortcode_ui_args() {
 
 	public static function setup_actions() {
 		add_action( 'init', 'Shortcake_Bakery\Shortcodes\Facebook::action_init_register_scripts' );
+		add_action( 'shortcode_ui_after_do_shortcode', function( $shortcode ) {
+			if ( false !== stripos( $shortcode, '[facebook' ) ) {
```

Comment:
@danielbachhuber Updated to use `get_shortcode_tag`. How would you test output to TinyMCE?